_commands/basics/uniq.md

```diff
@@ -2,23 +2,39 @@
 ---
 
 uniq
---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+------
+
+`uniq`, often used with  `sort`, allows a user to report, or filter, repeated lines that are adjacent in a file. Some common and useful flags include -d or -u for listing lines that are duplicated or that are unique, respectfully. As well as -c for counting the number of times a line was dupicated.
```

Comment:
The line from "Some common..." onward can be moved below the "`<!--more-->`" comment. That way the amount of content that appears on the reference page is minimal.